back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/registry/MicrometerRegistry.java

```diff
@@ -217,7 +217,7 @@ public SampleLog makeLog(String name, Date timestamp) {
 		Double totalThroughput = this.summaryRegistry.counter("count.total","sample",name).count() / (timeSinceFirstSampleCallInSeconds);
 		
 		return currentPeriodSummary == null ? null : new SampleLog(
-				Util.micrometerToOpenMetrics(name),
+				Util.makeOpenMetricsName(name),
```

Comment:
Is this related to the PR ? or should you upgrade your branch ?

Reply:
Actually, this branch was already merged to the develop branch. And I commited in the same merged branch, then i did the PR . I shouldn't do that ? Should I create a new branch ?